src/proposals/20250306/SparkEthereum_20250306.t.sol

```diff
@@ -0,0 +1,168 @@
+            baseVariableBorrowRate: 0.05e27,
+            variableRateSlope1:     0.15e27,
+            variableRateSlope2:     3e27,
+            // Borrowing configuration
+            borrowEnabled:          false,
+            stableBorrowEnabled:    false,
+            isolationBorrowEnabled: false,
+            siloedBorrowEnabled:    false,
+            flashloanEnabled:       false,
+            // Reserve configuration
+            ltv:                  65_00,
+            liquidationThreshold: 70_00,
+            liquidationBonus:     108_00,
+            reserveFactor:        15_00,
+            // Supply caps
+            supplyCap:    250,
+            supplyCapMax: 2500,
+            supplyCapGap: 250,
+            supplyCapTtl: 12 hours,
+            // Borrow caps
+            borrowCap:    0,
+            borrowCapMax: 0,
+            borrowCapGap: 0,
+            borrowCapTtl: 0,
+            // Isolation  and emode configurations
+            isolationMode:            false,
+            isolationModeDebtCeiling: 0,
+            liquidationProtocolFee:   10_00,
+            emodeCategory:            3
+        });
+
+        CollateralOnboardingTestParams memory tbtcConfigParams = CollateralOnboardingTestParams({
+            // General
+            symbol:       'tBTC',
+            tokenAddress:  Ethereum.TBTC,
+            tokenDecimals: 18,
+            oracleAddress: AGGOR_BTCUSD_ORACLE,
+            // IRM Params
+            optimalUsageRatio:      0.60e27,
+            baseVariableBorrowRate: 0,
+            variableRateSlope1:     0.04e27,
+            variableRateSlope2:     3e27,
+            // Borrowing configuration
+            borrowEnabled:          true,
+            stableBorrowEnabled:    false,
+            isolationBorrowEnabled: false,
+            siloedBorrowEnabled:    false,
+            flashloanEnabled:       true,
+            // Reserve configuration
+            ltv:                  65_00,
+            liquidationThreshold: 70_00,
+            liquidationBonus:     108_00,
+            reserveFactor:        20_00,
+            // Supply caps
+            supplyCap:    125,
+            supplyCapMax: 500,
+            supplyCapGap: 125,
+            supplyCapTtl: 12 hours,
+            // Borrow caps
+            borrowCap:    25,
+            borrowCapMax: 250,
+            borrowCapGap: 25,
+            borrowCapTtl: 12 hours,
+            // Isolation  and emode configurations
+            isolationMode:            false,
+            isolationModeDebtCeiling: 0,
+            liquidationProtocolFee:   10_00,
+            emodeCategory:            0
+        });
+
+        CollateralOnboardingTestParams[] memory newCollaterals = new CollateralOnboardingTestParams[](2);
+        newCollaterals[0] = lbtcConfigParams;
+        newCollaterals[1] = tbtcConfigParams;
+
+        _testCollateralOnboardings(newCollaterals);
+    }
+
+    function test_ETHEREUM_cbBtcEmode() public {
```

Comment:
Need to also test adding the BTC e-mode.

Reply:
Added